src/main/java/util/Validator.java

```diff
@@ -0,0 +1,36 @@
+package util;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+
+public class Validator {
+    public String checkLetter(BufferedReader reader) throws IOException {
+        String input = reader.readLine();
+        while (!(input.equals("r") || input.equals("b") || input.equals("s"))){
+            input = reader.readLine();
+        }
+        return input;
+    }
+
+    public String checkString(BufferedReader reader) throws IOException {
+        String input = reader.readLine();
+        while (input.isEmpty()){
+            input = reader.readLine();
+        }
+        return input;

```

Comment:
why don't checkFruitName() ? 
i suppose it's what this method intended to do
+ if you wanna validate fruit name it would be great to have regex, that you will use here, so name like this won't pass it "3242354235djsf"